wgpu-core/src/resource.rs

```diff
@@ -266,8 +266,8 @@ pub enum CreateTextureError {
     InvalidMipLevelCount(u32),
     #[error("The texture usages {0:?} are not allowed on a texture of type {1:?}")]
     InvalidUsages(wgt::TextureUsage, wgt::TextureFormat),
-    #[error("Feature {0:?} must be enabled to create a texture of type {1:?}")]
-    MissingFeature(wgt::Features, wgt::TextureFormat),
+    #[error("Texture format {0:?} can't be used")]
```

Comment:
nit:
```suggestion
    #[error("Texture format {0:?} can't be used due to a missing feature")]
```

Reply:
Since the actual `MissingFeatures` is the source for this, it should already say that a feature is missing. I haven't double-checked the final look of this, but we can follow-up and fix it.